src/classes/groundplaces.ts

```diff
@@ -0,0 +1,161 @@
+import {
+  AutoComplete,
+  StopGroup,
+  StopCluster,
+  StopGroupGpuid,
+  StopClusterGpuid,
+  Properties,
+  Filters,
+  GroundPlacesDiff,
+} from '../types';
+
+/**
+ * @description GroundPlaces business logic.
+ */
+export class GroundPlaces {
+  /**
+   * @description Returns a list of places.
+   * @param {string} query - Can be a name, a Gpuid, a unique name or other name.
+   * @param {Filters} filters - Filters with different options (StopGroup, StopCluster, Serviced, SegmentProvider).
+   * @returns {AutoComplete}
+   */
+  // @ts-ignore
+  public autocomplete(query: string, filters: Filters): AutoComplete {}
+
+  /**
+   * @description Create the stopGroup with the values given.
+   * @param {StopClusterGpuid} stopClusterGpuid - Ground place unique identifier of the stopCluster parent.
+   * @param {StopGroup} stopGroupInfos - StopGroup informations.
+   * @returns {StopGroup}
+   */
+  // @ts-ignore
+  public createStopGroup(stopClusterGpuid: StopClusterGpuid, stopGroupInfos: StopGroup): StopGroup {}
+
+  /**
+   * @description Create the stopCluster with the values given.
+   * @param {StopCluster} stopClusterInfos - StopCluster informations.
+   * @returns {StopCluster}
+   */
+  // @ts-ignore
+  public createStopCluster(stopClusterInfos: StopCluster): StopCluster {}
+
+  /**
+   * @description Update the stopGroup with the new values given.
+   * @param {StopGroupGpuid} stopGroupGpuid - Ground place unique identifier of a StopGroup.
+   * @param {Properties} propertiesToUpdate - Properties that need to be update.
```

Comment:
`Properties` seems a bit vague. Maybe something more specific like `StopGroupUpdateProperties` ou `StopGroupUpdateParams`.

Reply:
Renamed for `StopGroupProperties` and `StopClusterProperties`